betfairlightweight/filters.py

```diff
@@ -65,13 +67,31 @@ def streaming_order_filter(
     return {to_camel_case(k): v for k, v in args.items() if v is not None}
 
 
-def time_range(from_: str = None, to: str = None) -> dict:  # todo datetime conversion
+def time_range(
+    from_: Union[str, datetime.datetime] = None,
+    to: Union[str, datetime.datetime] = None,
+) -> dict:
     """
-    :param str from_:
-    :param str to:
+    :param Union[str, datetime.datetime] from_:
+    :param Union[str, datetime.datetime] to:
 
     :return: dict
     """
+
+    if isinstance(from_, datetime.datetime):
+        from_ = from_.isoformat()
+    elif isinstance(from_, datetime.date):
+        from_ = None
```

Comment:
Should we look to raise an error here?

Reply:
That's an idea. Or perhaps we could transform date into datetime, by setting the following times:

from_'s = 00:00:00
to's = 23:59:59

What's your opinion?

Reply:
Always wary of modifying data provided by the client, fine to go datetime to date but not sure the other way.